scripts/yggdrasil-iptunnel/yggdrasil-pre-setup

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Convert existing yggdrasil config to json from hjson
```

Comment:
Why is this only being done in `yggdrasil-iptunnel`? Why is it an issue in this case, and not with regular Yggdrasil?

Reply:
because you cannot cleanly edit hjson.  (IE jq does not work)